acceptance/openstack/ces/v1/alarms_test.go

```diff
@@ -0,0 +1,47 @@
+package v1
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/ces/v1/alarms"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestAlarms(t *testing.T) {
+	client, err := clients.NewCesV1Client()
+	th.AssertNoErr(t, err)
+
+	lim := 10
+	alarmsRes, err := alarms.ListAlarms(client, alarms.ListAlarmsRequest{
+		Limit: &lim,
+		Order: "desc",
+	}).Extract()
+
+	th.AssertNoErr(t, err)
+	th.AssertEquals(t, alarmsRes.MetaData.Count <= lim, true)
+
+	f := false
+	expAlarm := alarmsRes.MetricAlarms[0]
```

Comment:
it fails if no alarms listed

Reply:
i think you need to rework test

Reply:
maybe we can let zuul check once and see how it works?

Reply:
I already tested in my project, when no alarms found. Then next steps failed